src/common/it/formatter.ts

```diff
@@ -0,0 +1,203 @@
+/**
+ * Represents heading levels.
+ */
+export enum HeadingLevel {
+	/**
+	 * Represents a level 1 heading. (#)
+	 */
+	H1 = 1,
+	/**
+	 * Represents a level 2 heading. (##)
+	 */
+	H2 = 2,
+	/**
+	 * Represents a level 3 heading. (###)
+	 */
+	H3 = 3,
+}
+
+/**
+ * Represents timestamp styles.
+ */
+export enum TimestampStyle {
+	/**
+	 * Represents a short timestamp style.
+	 */
+	ShortTime = 't',
+	/**
+	 * Represents a long timestamp style.
+	 */
+	LongTime = 'T',
+	/**
+	 * Represents a short date style.
+	 */
+	ShortDate = 'd',
+	/**
+	 * Represents a long date style.
+	 */
+	LongDate = 'D',
+	/**
+	 * Represents a short time style.
+	 */
+	ShortDateTime = 'f',
+	/**
+	 * Represents a long time style.
+	 */
+	LongDateTime = 'F',
+	/**
+	 * Represents a relative time style.
+	 */
+	RelativeTime = 'R',
+}
+
+/**
+ * Represents a message link.
+ */
+type MessageLink = `https://discord.com/channels/${string}/${string}/${string}`;
+
+/**
+ * Represents a timestamp.
+ */
+type Timestamp = `<t:${number}:${TimestampStyle}>`;
+
+/**
+ * Represents a formatter utility for formatting content.
+ */
+export class Formatter {
```

Comment:
This does not need to be a class, it can be exported from the index as `* as Formatter`.

Reply:
> This does not need to be a class, it can be exported from the index as `* as Formatter`.

Yes, i can do that but i would need to change the naming scheme because colors.ts also have methods like bold, italic etc